### ISSUE TITLE
Class 'Symfony\\Component\\EventDispatcher\\Event' not found

### ISSUE THREAD

**Issue by mr-observer** [Codeception/Codeception#5886]
```
$codecept = new Codecept([
    'steps' => true,
    'verbosity' => 1,
]);

$codecept->run('acceptance');
```

**Class 'Symfony\\Component\\EventDispatcher\\Event' not found**

Codeception using deprecated Symfony components?
Any way to achieve results without downgrade the EventDispatcher or using exec?

**Comment by Naktibalda** [comment]
You are one of a very few people who use Codeception as a library, this isn't an officially supported way to use it.

Codeception creates class alias for `Symfony\Component\EventDispatcher\Event` if it doesn't exist:
https://github.com/Codeception/Codeception/blob/c30e2abe4af327d8be4fb23446ff740fec76fd60/shim.php#L40-L41

This file is loaded through a chain of files when codecept is executed using provided `codecept` script.

If you are using Codeception as a library, it is your responsibility to require shim.php or create class alias.
You are welcome to make pull request to improve usage of Codeception as a library.

**Comment by Naktibalda** [comment]
@mr-observer I think that I got your case covered too,
please test these branches:
```js
"codeception/codeception": "dev-abstract-test as 4.2.0",
"codeception/phpunit-wrapper": "dev-7.1-abstract-test as 7.9.99",
```

**Comment by Naktibalda** [comment]
This issue will be fixed in Codeception 5.0.0 #6067
